### ISSUE TITLE
Add a new category

### ISSUE THREAD

**Issue by redallica** [graphprotocol/mission-control-curator#156]
Add a new category for the Non Fungible tokens (NFT) projects. It might be related to Art and gaming, but it could also be much broader than that. I believe we need to think about and see what is the best way to categorize this type of projects

**Comment by yanivtal** [comment]
Yes there've been a few suggestions of where that could live. I'm still thinking about it and would maybe like to find a way to poll to see if wisdom of the crowds can help.
